src/pages/patientView/PatientViewPage.tsx

```diff
@@ -304,7 +304,7 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                                         sampleLabels={sampleManager.sampleLabels}
                                         sampleColors={sampleManager.sampleColors}
                                         sampleManager={sampleManager}
-                                        getWidth={()=>$(window).width()-140}
+                                        getWidth={()=>$(window).width() - ((patientViewPageStore.mutationData.result.length > 0) ? 140 : 40)}
```

Comment:
can we move this logic into the GenomicOverview class as a static method or something?